src/ontology/cl-edit.owl

```diff
@@ -7412,7 +7412,8 @@ SubClassOf(obo:CL_0000486 obo:CL_0002520)
 
 # Class: obo:CL_0000487 (oenocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:bf") Annotation(oboInOwl:hasDbXref "PMID:11171397") obo:IAO_0000115 obo:CL_0000487 "A large secretory cell found in clusters underlying the epidermis of the abdominal segments of larval abdominal segments.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:bf") Annotation(oboInOwl:hasDbXref "PMID:11171397") Annotation(oboInOwl:hasDbXref "PMID:24397521") Annotation(oboInOwl:hasDbXref "PMID:34208190") Annotation(oboInOwl:hasDbXref "https://www.merriam-webster.com/dictionary/oenocyte") obo:IAO_0000115 obo:CL_0000487 "A secretory cell of ectodermal origin in connection with the fat bodies and tracheae. This cell may have important functions in fatty acid and hydrocarbon metabolism.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24397521") Annotation(oboInOwl:hasDbXref "PMID:34208190") rdfs:comment obo:CL_0000487 "An oenocyte is highly variable in shape and size across species. This cell typically forms clusters. In adults, oenocytes tend to be smaller but more numerous, while in larvae they are larger and form smaller clusters. While oenocytes have been found in arachnids and crustaceans, they have been most abundantly described in insects.")
```

Comment:
Did not find resource to back up statement on variability in shape across species.

Reply:
> Did not find resource to back up statement on variability in shape across species.

[It is a bit subjective, but I considered circular and oval different shapes. However, it is true that it doesn't say the variability is across species (it can be in the same subject even)](https://www.mdpi.com/2075-4450/12/6/547#:~:text=Oenocytes%20are%20circular%20or%20oval%20cells%20associated)

Reply:
The more I think about it, the less I am comfortable with the idea of packing so many details in the definitions and comments of CL terms. There’s a high risk of ending up saying things that are simply not true across the taxonomic branch where the term is supposed to be valid. There’s a reason why the definitions I write for CL are often much less precise than the ones I would normally write for FBbt.

For example, there are oenocytes that are neither circular or ovoidal (https://doi.org/10.1146%2Fannurev-ento-011613-162056, Figure 1). And one can argue about the “typicality” of the organisation in clusters, when [Chapman suggests](https://doi.org/10.1017/CBO9780511818202.017) that there may be as many species in which they do form clusters as species in which they don’t (“In Lepidoptera and Orthoptera they form clusters in the body cavity, while in Homoptera, Hymenoptera and some Diptera they are dispersed and embedded in the fat body”).

Under the open world assumption, we should stick to only state what we know for sure is true and applicable to all the concerned species (which sometimes may be… “not much”). For the rest, better not to say anything rather than saying something wrong.

Reply:
(And yes, I know, the Chapman book is behind a paywall. Unfortunate, for sure, but I am not willing to skip on using reference books just because of that.)

Reply:
Thank you for your comment. I agree to make it more vague to avoid making the term class or species specific, when in this case it should describe the whole phylum. I still want to make a comment about the high variability of this cell.

> (And yes, I know, the Chapman book is behind a paywall. Unfortunate, for sure, but I am not willing to skip on using reference books just because of that.)

I agree that if an article/chapter is relevant, it should be used even if it is behind a paywall. The problem is that our organisation is not subscribed to as many publishing groups as some universities, and we cannot access them unless it is quoted or there is a screenshot in the ticket/PR (not sure if that suppose copyright infringements).

Reply:
> I still want to make a comment about the high variability of this cell.

Do you have access to the [Makki, Cinnamon, and Gould (2014)](https://doi.org/10.1146%2Fannurev-ento-011613-162056) paper? Between Figure 1 (which illustrates the morphological diversity) and the explicit statement that “the size, number, and anatomical locations of oenocytes all vary widely between insect species”, I would say it is enough to back up the high variability claim.

> The problem is that our organisation is not subscribed to as many publishing groups as some universities, and we cannot access them 

I understand, and I know I am in a privileged position here – I have never had access to so many journals than in the past two years, courtesy of my university. And since sometimes full-time access is done “transparently”, I may not even always realise that the link I provide is not accessible for everyone else. :(

> unless it is quoted or there is a screenshot in the ticket/PR (not sure if that suppose copyright infringements).

I would assume that quoting a paragraph fits under “fair use”, though obviously the usual disclaimer “I Am Not A Lawyer” applies.

Reply:
> Do you have access to the [Makki, Cinnamon, and Gould (2014)](https://doi.org/10.1146%2Fannurev-ento-011613-162056) paper? Between Figure 1 (which illustrates the morphological diversity) and the explicit statement that “the size, number, and anatomical locations of oenocytes all vary widely between insect species”, I would say it is enough to back up the high variability claim.

Sorry, I didn't express it very clear. I meant that I prefer to have a comment about the high variability than removing it altogether. After all, having this high variability is a characteristic.

>I understand, and I know I am in a privileged position here – I have never had access to so many journals than in the past two years, courtesy of my university. And since sometimes full-time access is done “transparently”, I may not even always realise that the link I provide is not accessible for everyone else. :(

It is not a problem, and I guess "the good" side of us having limited access is that we can provide more accessible literature. But again, I believe a relevant reference should be used even if it is behind a paywall.